src/app/fdctl/topology.c

```diff
@@ -184,7 +184,7 @@ fd_topo_workspace_fill( fd_topo_t *      topo,
       case FD_TOPO_LINK_KIND_QUIC_TO_VERIFY:
         /* The QUIC tile stashes some information in the dcache app region, this
             should probably be changed. */
-        dcache_app_sz = fd_quic_dcache_app_footprint( link->depth );
+        dcache_app_sz = fd_quic_dcache_app_footprint( ( link->depth + link->burst ) * FD_CHUNK_SZ );
```

Comment:
I don't understand why you multiply by FD_CHUNK_SZ here?

Reply:
I don't think that's right, the dcache_app_footprint function already does that multiplication internally.

Reply:
I'm not sure why I thought it was necessary. I've removed it, and it doesn't overflow in tests so far.